tests/unit/services/cookies-test.js

```diff
@@ -597,6 +610,19 @@ describe('CookiesService', function() {
         this.subject().write(COOKIE_NAME, value);
       });
 
+      it('writes the same name only once', function() {
+        let value1 = randomString();
+        let value2 = randomString();
+
+        this.subject().write(COOKIE_NAME, value1);
+        this.subject().write(COOKIE_NAME, value2);
+
+        const headers = this.fakeFastBoot.response.headers.getAll('set-cookie');
```

Comment:
Is `getAll` even needed? Could we just get the header from the `responseHeaders` local var directly?

Reply:
What I meant was that in the test `getAll` is just an extra step that is [implemented in the test itself](https://github.com/simplabs/ember-cookies/pull/195/files#diff-91737467d778614edb8e37696092b40bR445) and thus not really necessary. In fact, the check for `name` being present is also not really necessary as it is always called with a `name`?

Reply:
I don't think I understand what you're getting at,`getAll(name)` and `append(name, value)` are mocked out because they are parts of the real FastBootHeaders class that the [implementation](https://github.com/simplabs/ember-cookies/pull/195/files#diff-2d4817fecea72becc6e3e1b778dfc1d2R126) [uses](https://github.com/simplabs/ember-cookies/pull/195/files#diff-2d4817fecea72becc6e3e1b778dfc1d2R137).

The else case is of getAll is not used here so I'll remove it if you want, it just seemed odd to me to add half of a very simple method and leave a landmine for someone to step on later expecting it to work the same as the real class.